src/main/java/de/tum/in/www1/artemis/service/SubmissionService.java

```diff
@@ -163,19 +163,20 @@ public long countSubmissionLocks(long courseId) {
      * No manual result means that no user has started an assessment for the corresponding submission yet.
      * For exam exercises we should also remove the test run participations as these should not be graded by the tutors.
      *
-     * @param fileUploadExercise the exercise for which we want to retrieve a submission without manual result
+     * @param exercise the exercise for which we want to retrieve a submission without manual result
      * @param examMode flag to determine if test runs should be removed. This should be set to true for exam exercises
      * @return a submission without any manual result or an empty Optional if no submission without manual result could be found
      */
     @Transactional(readOnly = true)
```

Comment:
can we please remove this `@Transactional` here

Reply:
Done